### DISCUSSION TITLE
script config 是否對網絡訪問有使用限制?

### DISCUSSION THREAD

**Discussion by Mion354D** [zzzgydi/clash-verge#876] [Q&A]
發現在 script profile 中進行的網絡訪問請求均失敗了

scene: 我的伺服器供應商的訂閱連結不提供完整的 clash config, 僅提供諸如 IP 端口密碼等信息. 例如:
"{ip:XX.XX, port:XX, password:XX}"
想定期生成一份可用的 clash config. 之前使用 clash for windows, 其提供了前處理能力, 可以做到這一點.

在 clash-verge 中進行了嘗試:
1. 直接使用 remote config 訂閱連結, 之後在 script 中解析: 失敗, remote 直接不允許訂閱非 clash config
2. 寫一個 local config, 之後在 script 中 fetch 訂閱, 填入 local config 之中: 失敗, 在 script 中似乎無法發起請求

請教有任何解決辦法嗎?

**Comment by zzzgydi** [comment]
script只能执行简单的js代码，没有fetch那些
